.github/workflows/ci-notebooks.yml

```diff
@@ -15,27 +15,27 @@ on:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: ['3.10', '3.11']
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python
-      uses: actions/setup-python@v4
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Upgrade pip
-      run: |
-        pip install --upgrade pip
-    - name: Install the package locally
-      run: pip install -e .
-    - name: Install nbval
-      run: |
-        pip install nbval
-    - name: Run examples/tutorials
-      run: |
-        pytest --nbval-lax docs/irdl.ipynb --maxfail 1 -vv
-        pytest --nbval docs/tutorial.ipynb --maxfail 1 -vv
+      - uses: actions/checkout@v3
```

Comment:
Why do we have such a diff here?

Reply:
I have auto format on in vscode, and it did this when I edited the file. I would say that this is a change I would have wanted to make, it visually makes a bit more sense like this.

Reply:
Should this then maybe be a separate commit?

Reply:
Sure, happy to split it out

Reply:
no probs